Sources/AccountSDKIOSWeb/Client/Client.swift

```diff
@@ -115,14 +115,15 @@ public class Client: CustomStringConvertible {
         return authState
     }
     
-    private func createWebAuthenticationSession(withMFA: MFAType? = nil,
-                                                loginHint: String?,
-                                                extraScopeValues: Set<String> = [],
-                                                completion: @escaping LoginResultHandler) -> ASWebAuthenticationSession {
+    func createWebAuthenticationSession(withMFA: MFAType? = nil,
+                                        loginHint: String?,
```

Comment:
for consistency can we provide a default value for loginHint? Its already done for other method arguments